app/build.gradle

```diff
@@ -7,23 +7,29 @@ apply plugin: 'com.android.application'
 
 apply plugin: 'kotlin-android'
 
-apply plugin: 'kotlin-android-extensions'
-
 android {
-    compileSdkVersion 28
+    compileSdkVersion 31
     defaultConfig {
         applicationId "im.rah.nightwear"
-        minSdkVersion 25
-        targetSdkVersion 28
+        minSdkVersion 26
```

Comment:
@halbag do you know whether this bump was strictly required for the new feature? – I've had to make effort to target 25 here in the past to accommodate users stuck on early Wear 2.0.

See https://github.com/rahim/nightwear/issues/20, https://github.com/rahim/nightwear/pull/21

If it is strictly required, we'll need to weigh what proportion of users appear to still be using nightwear with these older watches against the benefit of the Tile view.

Reply:
I've just gotten this warning in the Studio, i guess Google Play requires now min SDK version 30. Not sure about it though. 
<img width="587" alt="Screenshot 2022-02-14 at 23 26 59" src="https://user-images.githubusercontent.com/9518042/153957045-81ad1fdc-275e-4fa3-b379-1e70fdb2729c.png">

Reply:
We can increase the `targetSdkVersion` and still support those older devices as long as the `minSdkVersion` isn't increased. I'm not sure what if any new problems that would create.

Reply:
I checked out the branch and tried building with
```
        minSdkVersion 25
        targetSdkVersion 30
```
I got the following error:
```
Manifest merger failed : uses-sdk:minSdkVersion 25 cannot be smaller than version 26 declared in library [androidx.wear.tiles:tiles-renderer:1.0.0] /Users/rahim/.gradle/caches/transforms-3/f7824c23f9cbb0d99504241bfe5d1b34/transformed/jetified-tiles-renderer-1.0.0/AndroidManifest.xml as the library might be using APIs not available in 25
	Suggestion: use a compatible library with a minSdk of at most 25,
		or increase this project's minSdk version to at least 26,
		or use tools:overrideLibrary="androidx.wear.tiles.renderer" to force usage (may lead to runtime failures)
```

Adding

```
<uses-sdk tools:overrideLibrary="androidx.wear.tiles.renderer"/>
```

to `app/src/debug/AndroidManifest.xml` appears to allow build to succeed again.